### ISSUE TITLE
Go to page number X

### ISSUE THREAD

**Issue by mladjan** [min/MNPageViewController#3]
Do you have idea what is the best way to implement feature to go to exact page?

**Comment by tonymillion** [comment]
Yeah, being able to directly set the displayed view controller is a _GLARING_ omission, seriously...
